include/solvers/postprocessors_smoothing.h

```diff
@@ -1,3 +1,18 @@
+/* ---------------------------------------------------------------------
+ *
+ * Copyright (C) 2019 -  by the Lethe authors
```

Comment:
Do you want to update the year?

Reply:
Maybe in another PR when we change the license everywhere